### PR TITLE
chore(docs): update info on router versions

### PR DESCRIPTION
**Pull request by EvanBacon**

### CODE REVIEW

File: docs/pages/router/reference/async-routes.mdx [hunk 1]
```diff
@@ -6,7 +6,7 @@ description: Learn how to speed up development with async bundling in Expo Route
 import { Terminal } from '~/ui/components/Snippet';
 import Video from '~/components/plugins/Video';
 
-> This guide requires `expo@49.0.0-alpha.3` or higher and `metro@0.76.3` &mdash; everything listed here is still experimental. To use these features, you may need to [use Expo CLI on `main`](https://github.com/expo/expo/tree/main/packages/%40expo/cli#contributing).
+> First available in Expo SDK 49 and Expo Router v2.
```
Comment (amandeepmittal) [inline docs/pages/router/reference/async-routes.mdx:9]:
```suggestion
> Available from Expo SDK 49 and Expo Router v2.
```

File: docs/pages/router/reference/static-rendering.mdx [hunk 2]
```diff
@@ -7,7 +7,7 @@ hideFromSearch: true
 import { Terminal } from '~/ui/components/Snippet';
 import { FileTree } from '~/ui/components/FileTree';
 
-> This guide requires `expo@49.0.0-alpha.3` or higher &mdash; everything listed here is still experimental. To use these features, you may need to [use Expo CLI on `main`](https://github.com/expo/expo/tree/main/packages/%40expo/cli#contributing).
+> First available in Expo SDK 49 and Expo Router v2.
```
Comment (amandeepmittal) [inline docs/pages/router/reference/static-rendering.mdx:10]:
```suggestion
> Available from Expo SDK 49 and Expo Router v2.
```

File: docs/pages/router/reference/async-routes.mdx (hunk 1 above)
Comment (amandeepmittal) [reply docs/pages/router/reference/async-routes.mdx:9]: Just a suggestion for the longevity of these docs.

File: docs/pages/router/reference/typed-routes.mdx [hunk 3]
```diff
@@ -7,7 +7,7 @@ import { Terminal } from '~/ui/components/Snippet';
 import { Tabs, Tab } from '~/ui/components/Tabs';
 import { FileTree } from '~/ui/components/FileTree';
 
-> Expo Router supports standard TypeScript out of the box. See [TypeScript](/guides/typescript/) guide for more information on how to set it up.
+> First available in Expo SDK 49 and Expo Router v2. Expo Router supports standard TypeScript out of the box. See [TypeScript](/guides/typescript/) guide for more information on how to set it up.
```
Comment (amandeepmittal) [inline docs/pages/router/reference/typed-routes.mdx:10]:
```suggestion
> Available from Expo SDK 49 and Expo Router v2. Expo Router supports standard TypeScript out of the box. See [TypeScript](/guides/typescript/) guide for more information on how to set it up.
```

**Review by amandeepmittal — APPROVED** [review]

File: docs/pages/router/reference/typed-routes.mdx [hunk 4]
```diff
@@ -7,7 +7,7 @@ import { Terminal } from '~/ui/components/Snippet';
 import { Tabs, Tab } from '~/ui/components/Tabs';
 import { FileTree } from '~/ui/components/FileTree';
 
-> Expo Router supports standard TypeScript out of the box. See [TypeScript](/guides/typescript/) guide for more information on how to set it up.
+> Available from Expo SDK 49 and Expo Router v2. Expo Router supports standard TypeScript out of the box. See [TypeScript](/guides/typescript/) guide for more information on how to set it up.
```
Comment (EvanBacon) [inline docs/pages/router/reference/typed-routes.mdx:10]:
```suggestion
> Available from Expo SDK 49 and Expo Router v2. Expo Router supports standard TypeScript out of the box. See the [TypeScript](/guides/typescript/) guide for more information on how to set it up.
```

**Pull request merged**